frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
if you're gonna put this here please override the default values

Reply:
completed